python/libraries/bimvee/geometry.py

```diff
@@ -114,6 +147,28 @@ def combineTwoQuaternions(q1, q2):
     qOut[1:4] = vOut
     return qOut
 
+'''
+Expects 
+    - poseDict in bimvee form {'ts', 'point', 'rotation' as above}
+    - translation as np array of x,y,z
+    - rotation as a np array of w,x,y,z (quaternion)
```

Comment:
might be useful to explicitly mention that the rotations are defined wrt local axes unlike the translations